sw/device/lib/dif/dif_lc_ctrl.c

```diff
@@ -66,6 +67,30 @@ dif_lc_ctrl_result_t dif_lc_ctrl_get_state(const dif_lc_ctrl_t *lc,
     case LC_CTRL_LC_STATE_STATE_VALUE_TEST_UNLOCKED3:
       *state = kDifLcCtrlStateTestUnlocked3;
       break;
+    case LC_CTRL_LC_STATE_STATE_VALUE_TEST_LOCKED3:
```

Comment:
mcyoung wonders... would this be better as a macro at this point?

Reply:
Could you elaborate? Do you mean having a macro that expands to each locked case block?

Reply:
I think that is what @mcy is referring to, as this is starting to look as a long configuration table.

Reply:
Ah I see. The issue we have is that the two "lookup tables" we have, implemented as switch blocks in dif_lc_ctrl_get_state() and dif_lc_ctrl_transition(), are slightly different. Specifically, there are a few states in dif_lc_ctrl_get_state() that don't exist in dif_lc_ctrl_transition(), i.e.: LC_CTRL_LC_STATE_STATE_VALUE_POST_TRANSITION, LC_CTRL_LC_STATE_STATE_VALUE_ESCALATE, and LC_CTRL_LC_STATE_STATE_VALUE_INVALID, since those states cannot be transitioned to from SW.